spec/features/cases/search_filter_journeys_spec.rb

```diff
@@ -251,11 +251,6 @@
 
       cases_search_page.filter_on_exemptions(common: %w{ s40 } )
 
-      cases_search_page.filter_tab_links.assigned_to_tab.click
-      cases_search_page.assigned_to_filter_panel.business_unit_search_term.set('foi')
-      cases_search_page.assigned_to_filter_panel.foi_responding_team_checkbox.click
-      cases_search_page.assigned_to_filter_panel.apply_filter_button.click
-
```

Comment:
@digitalali-moj Do you happen to recall why this stuff got removed?

Reply:
For everyone: Part of the ticket was to remove the assigned to filter.